src/model/sensor.ts

```diff
@@ -1,10 +1,31 @@
-const SensorTypes = ['TEMPERATURE_AND_AIR_HUMIDITY', 'WIND_SPEED', 'WIND_DIRECTION', 'LITTER_MOISTURE', 'PM2_5', 'CO2'];
+import { MapLocation } from './geography';
 
-type SensorType = (typeof SensorTypes)[number];
+export const SensorTypes = [
+  'TEMPERATURE_AND_AIR_HUMIDITY',
+  'WIND_SPEED',
+  'WIND_DIRECTION',
+  'LITTER_MOISTURE',
+  'PM2_5',
+  'CO2',
+];
 
-interface Sensor {
+export type SensorType = (typeof SensorTypes)[number];
+
+export interface Sensor {
   sensorId: number;
   sensorType: SensorType;
   location: MapLocation;
   timestamp: Date;
 }
+
+export const getDefaultSensor = (): Sensor => {
+  return {
+    sensorId: 0,
+    sensorType: 'CO2',
+    location: {
+      latitude: 0,
+      longitude: 0,
+    },
+    timestamp: new Date(Date.now()),
```

Comment:
It's enough to call `new Date()` - when no argument provided it creates the Date object with the current time